deepgrp/__main__.py

```diff
@@ -307,8 +308,28 @@ def train(args: argparse.Namespace, options: dgmodel.Options):
         with open(args.parameter, "r") as file:
             parameter = dgmodel.Options.from_toml(file)
         parameter.fromdict(options.todict())
-
-        raise NotImplementedError()  # TODO:  Add training function here.
+        logdir = args.logdir
+
+        # add _LOG.xxx functions for logging
+        # include a check if preprocessed?
+        # check if logdir exists?
+        # how to load data in?
+        # specific name for trained model?
+        # load data into np.ndarray
+        _LOG.info("Loading %s as training data", args.trainfile)
+        train_data = np.load(args.trainfile, allow_pickle=False)
+        _LOG.info("Loading %s as validation data", args.validfile)
+        val_data = np.load(args.valdata, allow_pickle=False)
+
```

Comment:
Here you get back a dictionary like object with "fwd" and "hash" keys. Hash is not used here. 
"fwd" contains the one-hot-encoded DNA sequence, which still needs some further preprocessing as done in 
Section 3.1 of [Training.ipynb](https://github.com/fhausmann/deepgrp/blob/master/notebooks/Training.ipynb).